src/SDK/Resource/ResourceInfo.php

```diff
@@ -122,4 +125,25 @@ public function getSchemaUrl(): ?string
     {
         return $this->schemaUrl;
     }
+
+    private function mergeWith(ResourceInfo $resource): ResourceInfo
+    {
+        $attributes = $this->attributes->toArray() + $resource->getAttributes()->toArray();
+
+        $schemaUrl = null;
+        if (null === $this->schemaUrl) {
```

Comment:
Please, do not use yoda style.
Please, use the accessor for this property here, since the accessor is used in the updatig resource and this will make it easier to read.

Reply:
I'm not sure that `php-cs-fixer` is configured properly to avoid yoda style.

Reply:
No it's not, as there is a lot of it in the code atm, which would needed to be changed (not sure if the fixer can handle this correctly).
However we have agreed in a meetig, that we'd rather want to avoid it in the future.